source/configuration_manual/service_configuration.rst

```diff
@@ -216,6 +220,11 @@ Auth master process connects to auth worker processes. It is mainly used by pass
 
    * **chroot** could also be set if possible.
 
+   * **service_count=0** counts the number of processed auth requests. This can be used to cycle the process after the specified number of auth requests (default is unlimited). The worker processes also stop after being idle for ``idle_kill`` seconds.
```

Comment:
I started thinking if the relation of the separate service_count lines maybe should somehow be clarified after these changes. Do we recommend using 0 instead of 1 now? The old line proposes 1, if I understand it correctly. Should we just replace the old line completely or mark it deprecated.

Reply:
better?